thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/onboard/tasks/FunctionCreationOnboardingTask.java

```diff
@@ -271,6 +271,7 @@ public void run() {
       alertConfig.setApplication(configuration.getString(ALERT_APPLICATION));
       alertConfig.setRecipients(alertRecipients);
       alertConfig.setCronExpression(configuration.getString(ALERT_CRON, DEFAULT_ALERT_CRON));
+      alertConfig.setActive(DEFAULT_IS_ACTIVE);
```

Comment:
Are you sure about this? This means any alert, even if backtesting fails, will start eating up cycles

Reply:
When do you think it is a proper place to activate the new alert?

Reply:
after it successfully completes backtesting, at least

Reply:
Thanks for the comment. It's fixed.